src/rpc/rawtransaction.cpp

```diff
@@ -1470,6 +1470,139 @@ UniValue sendrawtransaction(const UniValue &params, bool fHelp)
     return hashTx.GetHex();
 }
 
+void InputDebuggerToJSON(const CInputDebugger &input, UniValue &result)
+{
+    std::map<std::string, std::string>::iterator it;
+
+    result.pushKV("isValid", input.isValid);
+    UniValue uv_vdata(UniValue::VARR);
+    for (auto data : input.vData)
+    {
```

Comment:
Not such a big deal here but still should use a reference. There are also a couple of other places just below this section that need the same.

-auto data
+auto &data

Reply:
fixed